include/NovelRT/Graphics/CameraBlock.h

```diff
@@ -1,9 +1,9 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
-#ifndef NOVELRT_CAMERABLOCK_H
-#define NOVELRT_CAMERABLOCK_H
+#ifndef NOVELRT_GRAPHICS_CAMERABLOCK_H
+#define NOVELRT_GRAPHICS_CAMERABLOCK_H
 #include <glm/glm.hpp>
 
-namespace NovelRT {
+namespace NovelRT::Graphics {
   struct CameraBlock {
     glm::mat4 cameraMatrix;
```

Comment:
nit: This should probably be `GeoMatrix`. IIRC, we had decided `glm` should be an internal implementation detail for now.